sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosError.java

```diff
@@ -4,6 +4,7 @@
 package com.azure.cosmos.models;
 
 import com.azure.cosmos.implementation.Constants;
+import com.azure.cosmos.implementation.Resource;
```

Comment:
`CosmosError` is in models (public surface) - and is still extending internal types `Resource` and `JsonSerializable`. This is exposed using `CosmosClientException` which can lead to potential exposure of Jackson libraries.